.gitlab-ci.yml

```diff
@@ -38,3 +43,17 @@ build-images:
   artifacts:
     reports:
       dotenv: build.env
+
+deploy-images-dev:
+  stage: deploy-dev
+  # This resource group is configured with process_mode=oldest_first to make sure the pipelines are run serially.
+  resource_group: production
+  # Image built from https://gitlab.eng.vmware.com/calatrava/cd-infra/-/blob/main/images/ci-deploy/Dockerfile
+  image: harbor-repo.vmware.com/calatrava/ci-deploy-stage:v4
```

Comment:
why version is `v4`?

Reply:
Can we pass this as a CI variable as it has an internal reference?

Reply:
Yes!